src/components/Tooltip/Tooltip.tsx

```diff
@@ -1,25 +1,117 @@
 /* eslint-disable react/jsx-props-no-spreading */
-import React from "react";
-import PropTypes from "prop-types";
+import { ReactElement, CSSProperties, PureComponent, isValidElement } from "react";
 import classnames from "classnames";
+import { Modifier } from "react-popper";
 import isFunction from "lodash/isFunction";
 import Dialog from "../Dialog/Dialog";
-import { DialogPositions } from "../../constants/sizes";
-import { DIALOG_ANIMATION_TYPES } from "../../constants/AnimationTypes";
-import { TOOLTIP_JUSTIFY_TYPES, TOOLTIP_THEMES } from "./TooltipConstants";
+import { BASE_SIZES_WITH_NONE, DialogPosition } from "../../constants";
+import VibeComponentProps from "../../types/VibeComponentProps";
+import { TooltipArrowPosition, TooltipTheme } from "./TooltipConstants";
+import { HideShowEvent, JustifyType, AnimationType } from "../../constants/dialog";
 import "./Tooltip.scss";
 
+export interface TooltipProps extends VibeComponentProps {
+  style?: CSSProperties;
+  children: ReactElement;
+  content: string | ReactElement;
```

Comment:
@hadasfa i think that string is a ReactElement so no need to mention both

Reply:
tried to removed it but its throw errors:
<img width="992" alt="צילום מסך 2022-10-19 ב-13 41 00" src="https://user-images.githubusercontent.com/72390374/196669419-b882cf8a-5613-4f5f-b8e0-7afe6617cb72.png">

Reply:
so lets add a type for it, so we can reuse it